### PR TITLE
[linstor] Document kernel-headers installation

### PR DESCRIPTION
**Pull request by kvaps**
## Why do we need it, and what problem does it solve?

Kernel-headers required to compile DRBD kernel module

## What is the expected result?

Users get infromed about kernel-headers requirement

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: linstor
type: fix
summary: Document kernel-headers installation
impact_level: low
```

### CODE REVIEW

File: modules/031-linstor/docs/CONFIGURATION.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ data:
   linstorEnabled: "true"
 ```
 
+> Module requires kernel-headers be installed on all your nodes:
```
Comment (z9r5) [inline modules/031-linstor/docs/CONFIGURATION.md:12]:
```suggestion
> **Cation!** Module requires kernel-headers to be installed. Run the following command on all your nodes:
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/031-linstor/docs/CONFIGURATION.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ data:
...
+>
+> ```bash
+> # ubuntu/debian
+> apt install linux-headers-$uname -r()"
+> # centos
+> yum install kernel-devel-$(uname -r)
+> ```
```
Comment (z9r5) [inline modules/031-linstor/docs/CONFIGURATION.md:19]:
```suggestion
> **Cation!** Module requires kernel-headers to be installed. Run the following command on all your nodes.
>
> For Ubuntu/Debian:
>
> ```shell
> apt install linux-headers-$uname -r()"
> ```
>
> For CentOS:
>
> ```shell
> yum install kernel-devel-$(uname -r)
> ```
```

File: modules/031-linstor/docs/CONFIGURATION_RU.md [hunk 3]
```diff
@@ -9,6 +9,15 @@ data:
...
 ```
 
+> Модуль требует установленных заголовочных файлов для вашего ядра:
+>
+> ```bash
+> # ubuntu/debian
+> apt install linux-headers-$uname -r()"
+> # centos
+> yum install kernel-devel-$(uname -r)
+> ```
```
Comment (z9r5) [inline modules/031-linstor/docs/CONFIGURATION_RU.md:19]:
```suggestion
> **Внимание!** Модуль требует установленных заголовочных файлов для вашего ядра. Выполните следующую команду на всех узлах кластера
>
> Для Ubuntu/Debian:
>
> ```shell
> apt install linux-headers-$uname -r()"
> ```
>
> Для CentOS:
>
> ```shell
> yum install kernel-devel-$(uname -r)
> ```
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**